lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -63,9 +63,9 @@ class DetailsRenderer {
         return null;
 
       default: {
-        // @ts-ignore tsc thinks this unreachable, but ts-ignore for error message just in case.
-        const detailsType = details.type;
-        throw new Error(`Unknown type: ${detailsType}`);
+// Tsc thinks this is unreachable. But renderers should be forward compatible with new unexpected detail types.
```

Comment:
@exterkamp hates indentation, apparently

Reply:
ya he do

Reply:
rt, don't like it